nengo_spa/algebras/tvtb_algebra.py

```diff
@@ -81,6 +88,64 @@ def _get_sub_d(self, d):
             raise ValueError("Vector dimensionality must be a square number.")
         return sub_d
 
+    def create_vector(self, d, properties, *, rng=None):
```

Comment:
Looks like this method fully duplicates VTB except for the changes in string names and the sidedness of the `identity_element`? Is there a rationale for not reusing code here (and similar elsewhere)?

Reply:
https://github.com/nengo/nengo-spa/pull/271#discussion_r593914636